packages/aws-lambda/layer/bin/publish-layer.sh

```diff
@@ -58,19 +60,23 @@ cat <<EOF >> package.json
 }
 EOF
 
-echo "step 2/5: downloading latest packages"
+echo "step 3/7: downloading latest packages"
 npm install $PACKAGE_NAMES
 VERSION=$(jq -r .version node_modules/@instana/aws-lambda/package.json)
 echo "downloaded version $VERSION"
 rm -f package.json package-lock.json
 cd ..
 
-echo "step 3/5: creating local zip file with layer contents"
+echo "step 4/7: Add extension to layer"
+mkdir -p extensions
+cp ../include/instana-lambda-extension ./extensions
```

Comment:
I think we should rather copy the extension binary into this repository and commit it instead of using something that is in .gitignore. Updating the extension would then also be a commit that is visible in the history. When including this from a file that only exists locally we do not get reproducible builds. Also, the Lambda layer is built on the CI server so you would need to get the extension binary there somehow anyway.